src/components/Navbar.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Books from './Books';
+import Categories from './Categories';
+
+function Navbar() {
+  return (
+    <Routes>
+      <Route path="/" element={<Books />} />
+      <Route path="/" element={<Categories />} />
```

Comment:
- [ ] I'm not able to go to the categories route because you are using the wrong path for categories. You are using home path `"/"` for categories but it should be `"/categories"`. Please fix this issue.
  - Change the second route with this code `<Route path="/categroies" element={<Categories />} />`